pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -344,10 +344,6 @@ stdenv.mkDerivation (finalAttrs: {
       --replace /usr/local/lib/frei0r-1 ${frei0r}/lib/frei0r-1
     substituteInPlace doc/filters.texi \
       --replace /usr/local/lib/frei0r-1 ${frei0r}/lib/frei0r-1
-  '' + lib.optionalString withVulkan ''
-    # FIXME: horrible hack, remove for next release
-    substituteInPlace libavutil/hwcontext_vulkan.c \
-      --replace VK_EXT_VIDEO_DECODE VK_KHR_VIDEO_DECODE
```

Comment:
Have you tested whether ffmpeg_4 and 5 still work without the hack?

Reply:
I can confirm they both build. I checked before suggesting the fix.
At least they do on x86_64-linux, although I believe the platform makes no difference in this particular case.

Reply:
Yeah, I can also confirm in my x86_64-linux machine.